build.gradle

```diff
@@ -47,3 +51,16 @@ group 'com.palantir.websecurity'
 version gitVersion()
 
 sourceCompatibility = 1.8
+
+task resolveConfigurations {
```

Comment:
why not use the configuration resolver plugin? :'(

Reply:
#80 :)